common/djangoapps/util/models.py

```diff
@@ -62,3 +63,49 @@ def to_python(self, value):
             value = decompress_string(value)
 
         return value
+
+
+class SerializedChoicesField(models.CharField):
```

Comment:
General documentation for custom fields is here, it describes the `to_python` and `get_prep_value` overrides.

https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/